JSTests/stress/temporal-instant.js

```diff
@@ -285,6 +285,16 @@ shouldBe(`${Temporal.Instant.from('1976-11-18T15:23:30+00')}`, '1976-11-18T15:23
 shouldBe(`${Temporal.Instant.from('1976-11-18T15Z')}`, '1976-11-18T15:00:00Z');
 // ignores any specified calendar
 shouldBe(`${Temporal.Instant.from('1976-11-18T15:23:30.123456789Z[u-ca=discord]')}`, '1976-11-18T15:23:30.123456789Z');
+// unknown annotations are ignored
```

Comment:
Are there already equivalent tests in test262? I think so, but if not, we should add them

Reply:
Yes, there are equivalent tests:

- test/built-ins/Temporal/Instant/from/argument-string-calendar-annotation.js
- test/built-ins/Temporal/Instant/from/argument-string-time-zone-annotation.js
- test/built-ins/Temporal/Instant/from/argument-string-unknown-annotation.js